services/wfm/src/main/java/org/openkilda/wfm/topology/event/service/IslService.java

```diff
@@ -58,10 +58,14 @@ public IslService(TransactionManager transactionManager, RepositoryFactory repos
     public void createOrUpdateIsl(Isl isl, Sender sender) {
         createOrUpdateIsl(isl);
 
-        String reason = String.format("Create or update ISL: %s_%d-%s_%d. ISL status: %s",
-                isl.getSrcSwitch().getSwitchId(), isl.getSrcPort(),
-                isl.getDestSwitch().getSwitchId(), isl.getDestPort(), isl.getStatus());
-        sender.sendRerouteInactiveFlowsMessage(reason);
+        Optional<Isl> foundIsl = islRepository.findByEndpoints(isl.getSrcSwitch().getSwitchId(), isl.getSrcPort(),
+                isl.getDestSwitch().getSwitchId(), isl.getDestPort());
+        if (foundIsl.isPresent() && !foundIsl.get().isUnderMaintenance()) {
```

Comment:
Why shouldn't we try to reroute inactive flows if previously unknown link appeared?

Reply:
`UP. do nothing (even not issue reroute for flows in DOWN state)` #1548